src/services/branch-manager.js

```diff
@@ -12,7 +12,7 @@ const ERROR_MESSAGE_NO_PRODUCTION_OR_REMOTE_ENVIRONMENT = 'You cannot run branch
 const ERROR_MESSAGE_NO_REMOTE_ENVIRONMENT = 'You cannot run this command until this project has a remote non-production environment.';
 const ERROR_MESSAGE_BRANCH_DOES_NOT_EXIST = 'This branch doesn\'t exist.';
 const ERROR_MESSAGE_REMOVE_BRANCH_FAILED = 'Failed to delete branch.';
-const ERROR_MESSAGE_NOT_ADMIN_USER = 'You need the \'Admin\' role on this project to use branches.';
+const ERROR_MESSAGE_NOT_ADMIN_USER = 'You need the \'Admin\' or \'Developer\' role on this project to use branches.';
```

Comment:
The variable name can be also updated:
```suggestion
const ERROR_MESSAGE_ROLE = 'You need the \'Admin\' or \'Developer\' role on this project to use branches.';
```